pkg/operator/vspherecontroller/driver_starter.go

```diff
@@ -302,12 +210,12 @@ func WithLogLevelDaemonSetHook() csidrivernodeservicecontroller.DaemonSetHookFun
 	}
 }
 
-func WithConfigMapDaemonSetAnnotationHook(configMapName, namespace string, configMapInformer corev1informers.ConfigMapInformer) csidrivernodeservicecontroller.DaemonSetHookFunc {
+func WithSecretDaemonSetAnnotationHook(secretName, namespace string, secretInformer corev1informers.SecretInformer) csidrivernodeservicecontroller.DaemonSetHookFunc {
 	return func(opSpec *operatorapi.OperatorSpec, ds *appsv1.DaemonSet) error {
 		inputHashes, err := resourcehash.MultipleObjectHashStringMapForObjectReferenceFromLister(
-			configMapInformer.Lister(),
 			nil,
-			resourcehash.NewObjectRef().ForConfigMap().InNamespace(namespace).Named(configMapName),
+			secretInformer.Lister(),
+			resourcehash.NewObjectRef().ForSecret().InNamespace(namespace).Named(secretName),
```

Comment:
Strictly speaking, do we even need this anymore? We are not mounting the csi-config in daemonset anyways...

Reply:
I'm not sure; do you remember why we added this? The only ref I could find was this: https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/107/files#diff-1d5594c6562bc9ce4771e4821f10c8580165298d4067db516eb283385676e5efR122

Reply:
Added a comment, PTAL